### ISSUE TITLE
AttributeError: module 'keras.api._v2.keras.optimizers' has no attribute 'experimental'

### ISSUE THREAD

**Issue by diksha7869** [tensorflow/tensorflow#59469]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Bug

### Have you reproduced the bug with TF nightly?

Yes

### Source

source

### Tensorflow Version

'2.6.0'

### Custom Code

Yes

### OS Platform and Distribution

windows10

### Python version

3.9.12

### Current Behaviour?

```shell
A bug happened!
AttributeError: module 'keras.api._v2.keras.optimizers' has no attribute 'experimental'
```

### Standalone code to reproduce the issue

```shell
AttributeError                            Traceback (most recent call last)
Cell In [34], line 1
----> 1 trained_model = run_experiment()

Cell In [33], line 8, in run_experiment()
      3 filepath = "C:/Users/HP/video_classifier"
      4 checkpoint = keras.callbacks.ModelCheckpoint(
      5     filepath, save_weights_only=True, save_best_only=True, verbose=1
      6 )
----> 8 model = get_compiled_model()
      9 history = model.fit(
     10     X_train,
     11     y_train,
   (...)
     14     callbacks=[checkpoint],
     15 )
     17 model.load_weights(filepath)

Cell In [32], line 21, in get_compiled_model()
     16     model = keras.Model(inputs, outputs)
     17     lr_schedule = keras.optimizers.schedules.ExponentialDecay(
     18     initial_learning_rate=1e-3,
     19     decay_steps=10000,
     20     decay_rate=0.9)
---> 21     opt = tf.keras.optimizers.experimental.AdamW(
     22     learning_rate=lr_schedule,
     23     beta_1=0.9,
     24     beta_2=0.999,
     25     epsilon=1e-08,
     26     weight_decay = 1e-04
     27 )
     28     model.compile(
     29         optimizer= opt, loss="sparse_categorical_crossentropy", metrics=["accuracy"]
     30     )
     31     return model

AttributeError: module 'keras.api._v2.keras.optimizers' has no attribute 'experimental'
```

### Relevant log output

_No response_</details>

**Comment by sanjail3** [comment]
Hi, can I work on this issue, can anyone help me out with this

**Comment by diksha7869** [comment]
please

**Comment by sanjail3** [comment]
I am new to Open source

**Comment by diksha7869** [comment]
please help me out in this error as i am not getting why this error came?

**Comment by sanjail3** [comment]
It seems that you are trying to use the experimental module from keras.optimizers which is not available in this version of Tensorflow.
This module was introduced in Tensorflow 2.4 and it is not available in 2.6.0.
As the error message suggests, you might want to use the optimizers that are available in the tf.keras.optimizers module or you can upgrade your Tensorflow version to the latest version.

**Comment by tiruk007** [comment]
@diksha7869 
Could you please provide the complete code or Colab gist to replicate the issue reported here. 
Please try with the latest Tensorflow version which is 2.11 and let us know if the issue still persists.

Thank you!
